mLRS/Common/hal/esp-timer.h

```diff
@@ -17,17 +17,28 @@
 #define SYSTICK_DELAY_MS(x)       (uint16_t)(((uint32_t)(x)*(uint32_t)1000)/SYSTICK_TIMESTEP)
 
 
-// Two variables are required to avoid race with HAL_IncTick() when tasks may take longer than one tick
-volatile uint32_t doSysTask = 0; // Only changed in ISR; incremented to signal tick
-uint32_t doneSysTask = 0; // Never changed in ISR; incremented when (doSysTask != doneSysTask)
+// Two variables allow to avoid race with HAL_IncTick() when tasks may take longer than one tick
+uint32_t doSysTask_done = 0; // Never changed in ISR; incremented when (uwTick != doSysTask_done)
+volatile uint32_t uwTick = 0; // Only changed in ISR; incremented to signal tick
 
-volatile uint32_t uwTick = 0;
+void resetSysTask()
+{
+    doSysTask_done = uwTick;
+}
 
+volatile bool doSysTask()
+{
+    if (uwTick != doSysTask_done)
+        {
```

Comment:
formatting pl, 
if () {
}
and 4 spaces ident
dito below
:)

Reply:
I think I followed the existing formatting in the file which I strongly thinks is what we want to do.  I think a separate PR should be used if you want to reformat existing code or it gets hard to see what changed later.

Can you point out the specific area where I didn't use 4 spaces?

Reply:
I'm not sure where you have seen that formatting of if clauses in the code
if there are exaples somewhere Ive missedthem, but that's not the normal
pl change it

Reply:
Sorry.  I got hung up on the 4 spaces thing and missed that you were talking about just the if statement

Reply:
I didn't look at what you typed carefully enough and thought you meant the function declaration.  BTW,  Why aren't you consistent between if and functions?

Reply:
I think the dito below part also contributed to my misunderstanding.  Where below?